src/components/Slider/utils.tsx

```diff
@@ -0,0 +1,22 @@
+import type { ViewStyle } from 'react-native';
+import type { ComponentStylePropWithVariants } from '../../types';
+
+type CustomProp = {
+    thumbTintColor: string;
+    minimumTrackTintColor: string;
+    maximumTrackTintColor: string;
+
+    slider: ViewStyle;
```

Comment:
this is not necessary, slider style can directly go into the root instead of slider object